src/css/base.scss

```diff
@@ -1,7 +1,80 @@
 /* This is an example of using Sass, otherwise plain CSS works too*/
-$primary-background: radial-gradient(circle, rgba(63,94,251,1) 0%, rgba(252,70,107,1) 100%);
-;
+$primary-background: radial-gradient(circle, rgb(197, 134, 61) 0%, rgb(37, 69, 226) 100%);
```

Comment:
It's been recommended that we use the `hsla` colour format! `rgba`, works for now, but keep the `hsla` thing in mind?